workspaces/orchestrator/plugins/orchestrator-backend/src/service/DataIndexService.ts

```diff
@@ -270,7 +270,7 @@ export class DataIndexService {
     const graphQlQuery = buildGraphQlQuery({
       type: 'ProcessInstances',
       queryBody:
-        'id, processName, processId, businessKey, state, start, end, nodes { id }, variables, parentProcessInstance {id, processName, businessKey}',
+        'id, processName, processId, state, start, end, nodes { id }, variables, parentProcessInstance {id, processName, businessKey}',
```

Comment:
to understand: why removing businessKey?

Reply:
this is a different PR. it is a requirement by piotr.